### DISCUSSION TITLE
How to rename currently edited file?

### DISCUSSION THREAD

**Discussion by aisbergde** [gsantner/markor#2229] [Q&A] [answered]
I can't find, how to rename the currently edited file. Is it possible and I just can't find? Or isn't it possible? If not yet possible, could this be implemented?

And / Or something like:
"jump to the file in the file list and mark it for operations"
-> the effect should be a selected file witch can be then renamed.

**Comment by gsantner** [comment, marked as answer]
Renaming is only in the filemanager list.

**Reply by aisbergde** [reply]
Could we implement a way to jump from the currently edited file to the filemanager list? Maybe in the 3-dot-menu in the upper right, or maybe in the info window showing file details?

What often happens with me:
I use the search functions to find and edit a file, or I use some virtual file lists (favorites, shortly seen), or I use Markor as external editor. In all these cases I am inside the file editor and sometimes I want to rename the file or to remove it into another folder or just to make a copy. And now it would be very helpful, if I could jump directly into the filemanager. The file could be selected, to highlight it and to be able to do operations like copy, move or rename.

**Reply by guanglinn** [reply]
For your information, see [here](https://github.com/gsantner/markor/pull/2211#issuecomment-1936885580).

**Reply by aisbergde** [reply]
Maybe, it could solve some issues.
It wouldn't solve, when I access the file via external access and use Markor as editor.

**Reply by harshad1** [reply]
Such a file may not even be renamable

**Reply by aisbergde** [reply]
It could be not renameable. But if there would be a jump from files content to the file manager list, then this would be the job of the file manager to check, if it can be renamed or not. It could be something like VSC "reveal in file explorer" or similar actions in many programms.
